tests/unit/test_scraper.py

```
@@ -689,4 +721,412 @@
+        "https://example.com/3"
+    ]
+    
+    # 複数のキャッシュファイルを作成
+    for url in urls:
+        cache_manager.save_to_cache(url, [{'model': 'iPhone', 'price': 100000}])
+        
+    # キャッシュのクリア
+    cache_manager.clear_cache()
+    assert len(list(cache_manager.cache_dir.glob('*.json'))) == 0
+    
+def test_scrape_url_with_cache():
```

Ruff (0.8.2)

Redefinition of unused `test_scrape_url_with_cache` from line 262

(F811)